base/src/main/scala/data/DisjunctionFunctions.scala

```diff
@@ -1,10 +1,17 @@
 package scalaz
 package data
 
+import Disjunction.{\/, \/-, -\/}
+
 trait DisjunctionFunctions {
-  @inline def -\/[L](value: L): Disjunction[L, Nothing] = Disjunction.-\/(value)
-  @inline def \/-[R](value: R): Disjunction[Nothing, R] = Disjunction.\/-(value)
+  @inline def -\/[L](value: L): Disjunction[L, Nothing] = -\/(value)
```

Comment:
Variance annotations should never be used. Consequently, the use of `Nothing` is suspicious.